routes/recordOverview.js

```diff
@@ -3,51 +3,113 @@ var linz = require('linz'),
     async = require('async'),
     clone = require('clone');
 
+
 /* GET /admin/:model/:id/overview */
 var route = function (req, res, next) {
 
-    var locals = {
-        model: req.linz.model,
-        record: clone(req.linz.record.toObject({ virtuals: true})),
-        permissions: req.linz.model.linz.formtools.permissions,
-        formtools: req.linz.model.linz.formtools
-    };
+    function transformDslToOverview (dsl, cb) {
```

Comment:
Should move the function `transformDslToOverview` to `L6` so it's a local function outside of `route` to keep the route function smaller. This provides better visibility to the code.